### PR TITLE
Removed Old Jade files and added pug files.

### PR DESCRIPTION
**Pull request by bnandaku**
Conversion to Pug is complete. Check #94

### CODE REVIEW

File: views/widget/portal-widget.pug [hunk 1]
```diff
@@ -152,11 +140,6 @@ block content
                         input(id='rating' type="checkbox" name="rating" ng-model="widget.rating")
                         label(for='rating') #[span] &nbsp; Ask for rating after the call ends
                           .badge(data-toggle='tooltip' data-placement="top" title="Ask the user/caller to rate after the call ends") ?
-                      .form-group(analytics-on analytics-event="Ringback Tone" analytics-label="Ringback Tone")
```
Comment (san983) [inline views/widget/portal-widget.pug]: Bad merging? please put this back

File: views/widget/portal-widget.pug [hunk 2]
```diff
@@ -124,9 +115,6 @@ block content
                         label(for='show_branding') #[span] Show branding
                           .badge(data-toggle='tooltip' data-placement="top" title="Toggles Voxbone branding from button and popup") ?
 
-                      .form-group(analytics-on analytics-event="Extra Configuration" analytics-label="Custom CSS")
```
Comment (san983) [inline views/widget/portal-widget.pug]: Why was this removed too?

File: views/widget/portal-widget.pug [hunk 3]
```diff
@@ -69,21 +69,12 @@ block content
...
-                        type="text", name="server_auth_url",
-                        ng-model-options="{updateOn: 'blur'}"
-                        ng-model="widget.server_auth_url"
-                        ng-pattern="/^(http|https):.+$/"
-                        ng-blur="validateAuthUri(widget_form)",
-                        ng-focus="resetUriFlags()"
-                        placeholder="i.e. https://tomcat.server/servlet/auth"
-                        required
-                      )
+                      input.form-control(type="text", name="server_auth_url", ng-model-options="{updateOn: 'blur'}" ng-model="widget.server_auth_url" ng-pattern="/^(http|https):.+$/" ng-blur="validateAuthUri(widget_form)", ng-focus="resetUriFlags()" placeholder="i.e. https://tomcat.server/servlet/auth" required)
```
Comment (san983) [inline views/widget/portal-widget.pug]: Please unify criteria here on putting `,` separators here or not :)

File: views/widget/portal-widget.pug [hunk 4]
```diff
@@ -123,10 +114,6 @@ block content
                         input(id='show_branding' type='checkbox' name='show_branding' ng-model='widget.show_branding')
                         label(for='show_branding') #[span] Show branding
                           .badge(data-toggle='tooltip' data-placement="top" title="Toggles Voxbone branding from button and popup") ?
-
-                      .form-group(analytics-on analytics-event="Extra Configuration" analytics-label="Custom CSS")
```
Comment (emilianop11) [inline views/widget/portal-widget.pug]: I believe we have to keep this.
